src/ocean/text/convert/Formatter.d

```diff
@@ -381,9 +381,52 @@ private void handle (T) (T v, FormatInfo f, scope FormatterSink sf, scope ElemSi
     static if (is(T == typeof(null)))
         se("null", f);
 
-    // Cannot print enum member name in D1, so just print the value
-    else static if (is (T V == enum))
-        handle!(V)(v, f, sf, se);
+
+    // Print enum member name - If `switch statement` is available
+    else static if (is(T V == enum) && canSwitchOn!(V))
```

Comment:
```suggestion
    else static if (is(T V == enum) && canSwitchOn!(T))
```

Reply:
I knew V was an element.
So I used elements of the element.